File: CodeForces/Java/CF_467A_GeorgeAndAccommodation.java
// Problem: 467A. George and Accommodation
// Link: https://codeforces.com/problemset/problem/467/A

import java.util.Scanner;
class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    int rooms = scan.nextInt();
    int roomsAvailable = 0;
    for (int i=0; i<rooms;i++){
      int people = scan.nextInt();
      int capacity = scan.nextInt();
      if (capacity - people >=2){
        roomsAvailable++;
      }
    }
    scan.close();
    System.out.println(roomsAvailable);
  }
}

File: CodeForces/Python/CF_467A_GeorgeAndAccommodation.py
# Problem: 467A. George and Accommodation
# Link: https://codeforces.com/problemset/problem/467/A

rooms = int(input())
roomsAvailable = 0

for i in range(rooms):
  people, capacity = map(int,input().split())
  if (capacity - people >=2):
    roomsAvailable = roomsAvailable + 1
print(roomsAvailable)